builder/builder.go

```diff
@@ -39,6 +39,10 @@ func NewBuilder(pm *procmanager.ProcManager, debug bool, workspaceDir string) *B
 // RunTask executes a Task.
 func (b *Builder) RunTask(ctx context.Context, task *graph.Task) error {
 	for _, network := range task.Networks {
+		if network.SkipCreation {
+			log.Printf("skip creating network " + network.Name)
```

Comment:
Nit: keep consistent with existing log style
1) Use uppercase for the first letter. skip => Skip.
2) Add newline at the end of the log.
3) Use format `%s`, eg, "Skip creating network: %s\n", network.Name.